src/assets/icons/BookMark/BookMark.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const BookMark = () => (
```

Comment:
I think this should be called `Bookmark`, in English bookmark is a single word, not two
```suggestion
const Bookmark = () => (
```

Reply:
Done